gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ junitVersion = "1.1.5"
 espressoCore = "3.5.1"
 lifecycleRuntimeKtx = "2.8.1"
 activityCompose = "1.9.0"
-composeBom = "2023.08.00"
+composeBom = "2024.05.00"
```

Comment:
Don't update any versions.. it might cause build issues with incompatible versions.